service/elastigroup/providers/aws/aws.go

```diff
@@ -478,7 +486,7 @@ type Predictive struct {
 }
 
 type Strategy struct {
-	Risk                     *float64         `json:"risk,omitempty"`
+	Risk                     *int             `json:"risk,omitempty"`
```

Comment:
This change breaks the backward compatibility of the SDK. Is there a legitimate reason for the change?

Reply:
@liranp 
The reason was type mismach with the optimizer that was reflected eventually to TF, it is possible to fix on provider side and cast before sending.
Would it be a more preferable solution to you?
@Lironrad